api/openapi-spec/v3/api__v1_openapi.json

```diff
@@ -1595,6 +1595,14 @@
             "default": {},
             "description": "State holds details about the container's current condition."
           },
+          "user": {
+            "allOf": [
+              {
+                "$ref": "#/components/schemas/io.k8s.api.core.v1.ContainerUser"
+              }
+            ],
+            "description": "User represents user identitiy information of the first process in the container"
```

Comment:
typo: identity

Reply:
addressed in [5f0d761](https://github.com/kubernetes/kubernetes/pull/117842/commits/5f0d7613520d87745a229f8cc854bbf0e2e18221)